indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -347,6 +347,24 @@ def __table_args__(cls):
     # - reviews (AbstractReview.abstract)
 
     @property
+    def candidate_contrib_types(self):
+        contrib_types = []
+        for track in self.reviewed_for_tracks:
+            if self.get_track_reviewing_state(track) == AbstractReviewingState.positive:
+                review = next(x for x in self.reviews if x.track == track)
```

Comment:
add a second `None` default argument. getting a `StopIteration` from `next()` can result in awful-to-debug bugs